Index: os-openvino/CMakeLists.txt
===================================================================
--- os-openvino.orig/CMakeLists.txt
+++ os-openvino/CMakeLists.txt
@@ -152,7 +152,7 @@ add_subdirectory(thirdparty)
 add_subdirectory(openvino)
 build_ngraph()
 add_subdirectory(inference-engine)
-add_subdirectory(model-optimizer)
+#add_subdirectory(model-optimizer)
 #add_subdirectory(docs)
 add_subdirectory(tools)
 
