Index: os-openvino/inference-engine/src/mkldnn_plugin/perf_count.h
===================================================================
--- os-openvino.orig/inference-engine/src/mkldnn_plugin/perf_count.h
+++ os-openvino/inference-engine/src/mkldnn_plugin/perf_count.h
@@ -46,4 +46,5 @@ public:
 
 }  // namespace MKLDNNPlugin
 
-#define PERF(_counter) PerfHelper __helper##__counter (_counter->PerfCounter());
+#define PERF(_counter)
+// #define PERF(_counter) PerfHelper __helper##__counter (_counter->PerfCounter());
