diff --git a/cmake/developer_package/packaging.cmake b/cmake/developer_package/packaging.cmake
index cda7afd..b4f51b7 100644
--- a/cmake/developer_package/packaging.cmake
+++ b/cmake/developer_package/packaging.cmake
@@ -18,9 +18,9 @@ list(APPEND CMAKE_MODULE_PATH
         set(IE_CPACK_RUNTIME_PATH ${IE_CPACK_IE_DIR}/bin/${ARCH_FOLDER}/${CMAKE_BUILD_TYPE} PARENT_SCOPE)
         set(IE_CPACK_ARCHIVE_PATH ${IE_CPACK_IE_DIR}/lib/${ARCH_FOLDER}/${CMAKE_BUILD_TYPE} PARENT_SCOPE)
     else()
-        set(IE_CPACK_LIBRARY_PATH ${IE_CPACK_IE_DIR}/lib/${ARCH_FOLDER} PARENT_SCOPE)
-        set(IE_CPACK_RUNTIME_PATH ${IE_CPACK_IE_DIR}/lib/${ARCH_FOLDER} PARENT_SCOPE)
-        set(IE_CPACK_ARCHIVE_PATH ${IE_CPACK_IE_DIR}/lib/${ARCH_FOLDER} PARENT_SCOPE)
+        set(IE_CPACK_LIBRARY_PATH ${CMAKE_INSTALL_LIBDIR} PARENT_SCOPE)
+        set(IE_CPACK_RUNTIME_PATH ${CMAKE_INSTALL_LIBDIR} PARENT_SCOPE)
+        set(IE_CPACK_ARCHIVE_PATH ${CMAKE_INSTALL_LIBDIR} PARENT_SCOPE)
     endif()
 endfunction()
 
diff --git a/inference-engine/src/inference_engine/CMakeLists.txt b/inference-engine/src/inference_engine/CMakeLists.txt
index f012a03..3c8cde7 100644
--- a/inference-engine/src/inference_engine/CMakeLists.txt
+++ b/inference-engine/src/inference_engine/CMakeLists.txt
@@ -261,7 +261,7 @@ endif()
 ie_cpack_add_component(core REQUIRED DEPENDS ${core_components})
 ie_cpack_add_component(core_dev REQUIRED core ngraph_dev)
 
-install(DIRECTORY "${IE_MAIN_SOURCE_DIR}/include" DESTINATION ${IE_CPACK_IE_DIR}
+install(DIRECTORY "${IE_MAIN_SOURCE_DIR}/include/" DESTINATION "include/openvino"
         COMPONENT core_dev)
 
 install(TARGETS ${TARGET_NAME} EXPORT InferenceEngineTargets
@@ -270,7 +270,7 @@ install(TARGETS ${TARGET_NAME} EXPORT InferenceEngineTargets
         LIBRARY DESTINATION ${IE_CPACK_LIBRARY_PATH} COMPONENT core)
 
 install(FILES $<TARGET_FILE_DIR:${TARGET_NAME}>/plugins.xml
-        DESTINATION ${IE_CPACK_RUNTIME_PATH}
+        DESTINATION ${IE_CPACK_RUNTIME_PATH}/openvino/inference-plugins/
         COMPONENT core)
 
 # Install cmake scripts
@@ -307,5 +307,5 @@ install(TARGETS ${TARGET_NAME}
 install(FILES "${CMAKE_BINARY_DIR}/share/InferenceEngineConfig.cmake"
               "${CMAKE_BINARY_DIR}/InferenceEngineConfig-version.cmake"
               "${InferenceEngine_SOURCE_DIR}/cmake/ie_parallel.cmake"
-        DESTINATION ${IE_CPACK_IE_DIR}/share
+        DESTINATION ${IE_CPACK_LIBRARY_PATH}/cmake/InferenceEngine/
         COMPONENT core_dev)
diff --git a/ngraph/CMakeLists.txt b/ngraph/CMakeLists.txt
index 99f32a6..4827ac9 100644
--- a/ngraph/CMakeLists.txt
+++ b/ngraph/CMakeLists.txt
@@ -183,10 +183,10 @@ endif()
 message(STATUS "Installation directory: ${CMAKE_INSTALL_PREFIX}")
 
 # Destinations
-set(NGRAPH_INSTALL_LIB "deployment_tools/ngraph/${CMAKE_INSTALL_LIBDIR}")
-set(NGRAPH_INSTALL_INCLUDE "deployment_tools/ngraph/${CMAKE_INSTALL_INCLUDEDIR}")
-set(NGRAPH_INSTALL_DOC "deployment_tools/ngraph/${CMAKE_INSTALL_DOCDIR}")
-set(NGRAPH_INSTALL_BIN "deployment_tools/ngraph/${CMAKE_INSTALL_BINDIR}")
+set(NGRAPH_INSTALL_LIB "${CMAKE_INSTALL_LIBDIR}")
+set(NGRAPH_INSTALL_INCLUDE "${CMAKE_INSTALL_INCLUDEDIR}")
+set(NGRAPH_INSTALL_DOC "${CMAKE_INSTALL_DOCDIR}")
+set(NGRAPH_INSTALL_BIN "${CMAKE_INSTALL_BINDIR}")
 
 #-----------------------------------------------------------------------------------------------
 # Compile Flags for nGraph...
@@ -272,7 +272,7 @@ if (NGRAPH_EXPORT_TARGETS_ENABLE)
         install(EXPORT ngraphTargets
             FILE ngraphTargets.cmake
             NAMESPACE ngraph::
-            DESTINATION "deployment_tools/ngraph/cmake"
+            DESTINATION ${NGRAPH_INSTALL_LIB}/cmake/ngraph
             COMPONENT ngraph_dev)
     endif()
 
@@ -286,7 +286,7 @@ if (NGRAPH_EXPORT_TARGETS_ENABLE)
 
     install(FILES ${CMAKE_CURRENT_BINARY_DIR}/ngraphConfig.cmake
         ${CMAKE_CURRENT_BINARY_DIR}/ngraphConfigVersion.cmake
-        DESTINATION "deployment_tools/ngraph/cmake"
+        DESTINATION ${NGRAPH_INSTALL_LIB}/cmake/ngraph
         COMPONENT ngraph_dev)
 endif()
 
diff --git a/cmake/developer_package/plugins/plugins.cmake b/cmake/developer_package/plugins/plugins.cmake
index 52b033d..ae06a02 100644
--- a/cmake/developer_package/plugins/plugins.cmake
+++ b/cmake/developer_package/plugins/plugins.cmake
@@ -123,8 +123,8 @@ function(ie_add_plugin)
         ie_cpack_add_component(${install_component} REQUIRED DEPENDS core)
 
         install(TARGETS ${IE_PLUGIN_NAME}
-            RUNTIME DESTINATION ${IE_CPACK_RUNTIME_PATH} COMPONENT ${install_component}
-            LIBRARY DESTINATION ${IE_CPACK_RUNTIME_PATH} COMPONENT ${install_component})
+            RUNTIME DESTINATION ${IE_CPACK_RUNTIME_PATH}/openvino/inference-plugins COMPONENT ${install_component}
+            LIBRARY DESTINATION ${IE_CPACK_LIBRARY_PATH}/openvino/inference-plugins COMPONENT ${install_component})
     endif()
 endfunction()
 
diff --git a/inference-engine/ie_bridges/c/src/CMakeLists.txt b/inference-engine/ie_bridges/c/src/CMakeLists.txt
index 223b635..adecba5 100644
--- a/inference-engine/ie_bridges/c/src/CMakeLists.txt
+++ b/inference-engine/ie_bridges/c/src/CMakeLists.txt
@@ -40,5 +40,5 @@ install(TARGETS ${TARGET_NAME}
         LIBRARY DESTINATION ${IE_CPACK_LIBRARY_PATH} COMPONENT core_c)
 
 install(DIRECTORY ${InferenceEngine_C_API_SOURCE_DIR}/include/
-        DESTINATION ${IE_CPACK_IE_DIR}/include
+        DESTINATION "include/openvino"
         COMPONENT core_c_dev)
