Index: os-openvino/thirdparty/ocl/icd_loader/CMakeLists.txt
===================================================================
--- os-openvino.orig/thirdparty/ocl/icd_loader/CMakeLists.txt
+++ os-openvino/thirdparty/ocl/icd_loader/CMakeLists.txt
@@ -60,7 +60,7 @@ include(JoinPaths)
 
 include(CheckFunctionExists)
 check_function_exists(secure_getenv HAVE_SECURE_GETENV)
-check_function_exists(__secure_getenv HAVE___SECURE_GETENV)
+#check_function_exists(__secure_getenv HAVE___SECURE_GETENV)
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/loader/icd_cmake_config.h.in
     ${CMAKE_CURRENT_BINARY_DIR}/icd_cmake_config.h)
 
