--- a/thirdparty/CMakeLists.txt
+++ b/thirdparty/CMakeLists.txt
@@ -12,5 +12,4 @@
     add_subdirectory(xbyak EXCLUDE_FROM_ALL)
     openvino_developer_export_targets(COMPONENT openvino_common TARGETS xbyak)
 endif()
-add_subdirectory(zlib EXCLUDE_FROM_ALL)
 add_subdirectory(cnpy EXCLUDE_FROM_ALL)
--- a/inference-engine/CMakeLists.txt
+++ b/inference-engine/CMakeLists.txt
@@ -72,12 +72,6 @@
 
 ie_cpack_add_component(cpp_samples DEPENDS core)
 
-install(DIRECTORY ../thirdparty/zlib
-        DESTINATION ${IE_CPACK_IE_DIR}/samples/cpp/thirdparty
-        COMPONENT cpp_samples
-        USE_SOURCE_PERMISSIONS
-        PATTERN .clang-format EXCLUDE)
-
 install(DIRECTORY ../thirdparty/cnpy
         DESTINATION ${IE_CPACK_IE_DIR}/samples/cpp/thirdparty
         COMPONENT cpp_samples
