diff --git a/inference-engine/cmake/features.cmake b/inference-engine/cmake/features.cmake
index 2c2c891..7476fb2 100644
--- a/inference-engine/cmake/features.cmake
+++ b/inference-engine/cmake/features.cmake
@@ -73,6 +73,8 @@ set(IE_EXTRA_MODULES "" CACHE STRING "Extra paths for extra modules to include i
 ie_dependent_option(ENABLE_TBB_RELEASE_ONLY "Only Release TBB libraries are linked to the Inference Engine binaries" ON "THREADING MATCHES TBB;LINUX" OFF)
 
 ie_option (USE_SYSTEM_PUGIXML "use the system copy of pugixml" OFF)
+
+ie_option(USE_SYSTEM_ADE "use the system copy of ade" OFF)
 
 ie_option (ENABLE_CPU_DEBUG_CAPS "enable CPU debug capabilities at runtime" OFF)
 
diff --git a/inference-engine/thirdparty/CMakeLists.txt b/inference-engine/thirdparty/CMakeLists.txt
index fa2a4d0..6689b69 100644
--- a/inference-engine/thirdparty/CMakeLists.txt
+++ b/inference-engine/thirdparty/CMakeLists.txt
@@ -61,14 +62,23 @@ else()
     target_include_directories(pugixml INTERFACE "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/pugixml/src>")
 endif()
 
-add_subdirectory(ade EXCLUDE_FROM_ALL)
+if (USE_SYSTEM_ADE)
+    find_package(ade REQUIRED)
+else()
+    add_subdirectory(ade EXCLUDE_FROM_ALL)
+    set_target_properties(ade PROPERTIES folder thirdparty)
+endif()
 add_subdirectory(fluid/modules/gapi)
 
-set_target_properties(ade fluid PROPERTIES FOLDER thirdparty)
+set_target_properties(fluid PROPERTIES FOLDER thirdparty)
 
 # developer package
 
-openvino_developer_export_targets(COMPONENT openvino_common TARGETS ade fluid)
+openvino_developer_export_targets(COMPONENT openvino_common TARGETS fluid)
+
+if (NOT USE_SYSTEM_ADE)
+    openvino_developer_export_targets(COMPONENT openvino_common TARGETS ade)
+endif()
 
 if (NOT USE_SYSTEM_PUGIXML)
     set_target_properties(pugixml PROPERTIES FOLDER thirdparty)
