diff --git a/inference-engine/cmake/features.cmake b/inference-engine/cmake/features.cmake
index 2c2c891..7476fb2 100644
--- a/inference-engine/cmake/features.cmake
+++ b/inference-engine/cmake/features.cmake
@@ -75,6 +75,8 @@ set(IE_EXTRA_MODULES "" CACHE STRING "Extra paths for extra modules to include i
 ie_option (USE_SYSTEM_PUGIXML "use the system copy of pugixml" OFF)
 
 ie_option(USE_SYSTEM_ADE "use the system copy of ade" OFF)
+
+ie_option (USE_SYSTEM_XBYAK "use the system copy of xbyak" OFF)
 
 ie_option (ENABLE_CPU_DEBUG_CAPS "enable CPU debug capabilities at runtime" OFF)

diff --git a/thirdparty/CMakeLists.txt b/thirdparty/CMakeLists.txt
index 2c6b6564..8f357619 100644
--- a/thirdparty/CMakeLists.txt
+++ b/thirdparty/CMakeLists.txt
@@ -4,7 +4,13 @@
 
 add_subdirectory(ittapi)
 add_subdirectory(itt_collector)
-add_subdirectory(xbyak EXCLUDE_FROM_ALL)
+if (USE_SYSTEM_XBYAK)
+    find_package(xbyak REQUIRED)
+    add_library(xbyak INTERFACE IMPORTED)
+    set_target_properties(xbyak PROPERTIES IMPORTED_GLOBAL TRUE INTERFACE_LINK_LIBRARIES xbyak::xbyak)
+else()
+    add_subdirectory(xbyak EXCLUDE_FROM_ALL)
+    openvino_developer_export_targets(COMPONENT openvino_common TARGETS xbyak)
+endif()
 add_subdirectory(zlib EXCLUDE_FROM_ALL)
 add_subdirectory(cnpy EXCLUDE_FROM_ALL)
-openvino_developer_export_targets(COMPONENT openvino_common TARGETS xbyak)
