diff --git a/ngraph/CMakeLists.txt b/ngraph/CMakeLists.txt
index 99f32a6..b0991d6 100644
--- a/ngraph/CMakeLists.txt
+++ b/ngraph/CMakeLists.txt
@@ -99,6 +99,7 @@ option(NGRAPH_USE_PROTOBUF_LITE "Compiles and links with protobuf-lite" FALSE)
 
 if (NGRAPH_ONNX_IMPORT_ENABLE)
     option(NGRAPH_USE_SYSTEM_PROTOBUF "Use system provided Protobuf shared object" OFF)
+    option(NGRAPH_USE_SYSTEM_ONNX "Use system provided onnx shared object" FALSE)
 endif()
 if(NGRAPH_ONNX_EDITOR_ENABLE AND NOT NGRAPH_ONNX_IMPORT_ENABLE)
     message(FATAL_ERROR "ONNX Editor component requires ONNX Importer. Set NGRAPH_ONNX_IMPORT_ENABLE=ON.")
@@ -314,7 +315,11 @@ if (NGRAPH_ONNX_IMPORT_ENABLE)
     endif()
 
     # target onnx_proto will be shared lib, onnx static
-    include(cmake/external_onnx.cmake)
+    if (NOT NGRAPH_USE_SYSTEM_ONNX)
+        include(cmake/external_onnx.cmake)
+    else()
+        find_package(ONNX 1.7.0 REQUIRED)
+    endif()
     if (TARGET ext_protobuf)
         add_dependencies(onnx ext_protobuf)
     endif()
