diff --git a/inference-engine/cmake/templates/InferenceEngineConfig.cmake.in b/inference-engine/cmake/templates/InferenceEngineConfig.cmake.in
index 261edbf3..61d21ba7 100644
--- a/inference-engine/cmake/templates/InferenceEngineConfig.cmake.in
+++ b/inference-engine/cmake/templates/InferenceEngineConfig.cmake.in
@@ -87,7 +87,7 @@ set(IE_PACKAGE_PREFIX_DIR "${PACKAGE_PREFIX_DIR}")
 
 set(THREADING "@THREADING@")
 if(THREADING STREQUAL "TBB" OR THREADING STREQUAL "TBB_AUTO" AND NOT TBB_FOUND)
-    set_and_check(_tbb_dir "@PACKAGE_IE_TBB_DIR@")
+    set(_tbb_dir "@PACKAGE_IE_TBB_DIR@")
     _ie_find_dependency(TBB
                         COMPONENTS tbb tbbmalloc
                         CONFIG
